### ISSUE TITLE
Help users better determine if cases returned in MUR search are relevant

### ISSUE THREAD

**Issue by adrianwebb** [18F/fec-eregs#296]
Comment: It would be helpful if a user was able to pull up and scroll through all “matches” to easily see if a case was relevant

**Comment by adrianwebb** [comment]
See all of the highlights of the document rather than just a few (Tony)

**Comment by nickykrause** [comment]
Dropping some notes in here, which I will clarify later:

Issues:

**Understanding where search terms appear in MUR docs**
Users want to see where their search terms match in specific MUR docs. 
- Current functionality is set up to combine the searchable text from all docs within a MUR. The result of this approach is that we can't determine which individual doc produced a search term match, and we can't prioritize specific doc types in the search. We could address that constraint for the existing MURs (which would be a tech lift), but not for archived MURs

**Understanding how relevance works**
Users don't understand how the documents are ordered (currently: by relevance). This may be related to their inability to assess how well the documents are matching on their terms (above), as well as the perceived over-inclusiveness or under-inclusiveness of results (see issue: #293 )

**Comment by jenniferthibault** [comment]
Carrying over one additional point from #297 before I close it:

**Ability to refine or narrow down a set of results**
> The problem seems to be that users don't have a clear sense of how the search results are determined, including how the display order works. Currently, the ordering of results is based on "relevance," but the users seem unable to assess whether or not the automated relevance calculation is indeed working (and how).

> The users' proposed solution is thus to return the results in chronological order, as well as to be able to search particular types of documents.

**Comment by jenniferthibault** [comment]
Since it's wasn't called out as a priority through FEC feedback in https://github.com/18F/fec-eregs/issues/289 AND since this involves possible research that won't be taken on this sprint, I've moved this into the MUR Refinement Epic instead of the First Release Epic.

**Comment by nickykrause** [comment]
Also pulling this from the user feedback on MURs, which can be found in [this document](https://docs.google.com/document/d/1eVFGVEDJEOESyjOptWO58uYJkg7NB26xF6rK9fCQE6E/edit#)

>It may help limit over-inclusiveness if searches pulled from only OGC prepared documents such as the First General Counsel’s Report, General Counsel’s Reports, Conciliation Agreement, General Counsel’s Brief and Commission-prepared documents such as Statements of Reason. 

As Porta pointed out: 
> This is possible but ONLY for current MURs. We don't have this information for archived MURs (as it's a single PDF for archived MURs).

**Comment by nickykrause** [comment]
I have been working through all the interconnected threads outlined above, and I have attempted to synthesize everything in the explanation below. This explanation is mostly for me, so that I can get a handle on what I think we should do next, research-wise, but I am posting it here in case it is of interest to others.

**TL;DR**: I think we should schedule a series of 45-min, individual, facilitated interviews with expert users at the FEC to address some of the outstanding questions we have about the MUR search experience, which will provide us with insights as to (a) how we can proceed with improving the users' ability to gauge relevance, (b) whether or not certain complex technical changes are warranted, and (c) how we might proceed with search refinement for MURs (as we did with AOs).

In the notes above, we acknowledged that users are interested in seeing exactly which documents within a MUR have produced their search term matches. 

Users requested this functionality because they felt it would help them to identify specific documents within a MUR that would be useful to read in more detail. The process of "diving deeper" into a MUR (by looking at specific documents that have keyword matches) is what helps the users to decide which of the MURs in their search results are most relevant to them. 

As we also noted above, the current MURs search functionality is not set up to enable the display of keyword matches by document type:
> Current functionality is set up to combine the searchable text from all docs within a MUR. The result of this approach is that we can't determine which individual doc produced a search term match, and we can't prioritize specific doc types in the search. We could address that constraint for the existing MURs (which would be a tech lift), but not for archived MURs

**A few questions arise from this situation:**
- **How important is this feature to the users? Do they feel it is essential to their work?** 
    - My hypothesis is that users would deem this essential or highly desirable.
    - If this hypothesis is true, then it would probably justify the technical lift of adjusting the search functionality. However...
- **Given that we can't address this problem for the archived MURs, we have another question: Is this feature only desirable to users in the event that it is all-inclusive?** Meaning: If only some of the results (the non-archived MURs) are able to display search term matches by document type, does that drastically undermine the utility of this feature for users?
    - My hypothesis is that users would still want this feature, even if it can't include archived MURs.

Next, we know that users have requested the ability to search (or refine their search) by document type, which, again, is not possible given the current state of the MUR search functionality. A few questions here:

- **Why are users interested in searching/refining by document type?** Is it because they are seeking a solution to the above-cited issue (seeing keyword matches by doc type), or is it because certain document types are typically of greater interest than others (similar to what we found with Advisory Opinions)? Or, perhaps it is both, or something else entirely...?
    - My hypothesis is that users are motivated to search by document type for both of the reasons mentioned above.
    - Given that users have requested more than one feature which would require a reconfiguration of the search design (to enable document-specific searching), we have further justification for the technical effort involved in searching non-archived MURs by doc type. But, again...
- **Is this feature only desirable if it is all-inclusive, or would it be okay for users if this document-specific searching only addressed non-archived MURs?**
    - I have no hypothesis here

Beyond the above issues, which influence a user's ability to parse search results and determine MUR relevance, we also suspect that users do not have a clear understanding of how their search results are being sorted (i.e., how they are ordered on the page, from top to bottom):

> The problem seems to be that users don't have a clear sense of how the search results are determined, including how the display order works. Currently, the ordering of results is based on "relevance," but the users seem unable to assess whether or not the automated relevance calculation is indeed working (and how).

> The users' proposed solution is thus to return the results in chronological order, as well as to be able to search particular types of documents.

 This issue also seems to affect their ability to determine the relevance of a MUR.

Some questions here:
- **What do users think of the current results sorting?**
- **Do they recognize that the results are sorted by relevance?**
    - How do they think this relevance sort is working? 
    - Do they seem to trust the relevance calculation?**
- **What preferences do users have about how their results are sorted?**
    - Is a reverse chronological sort indeed a strong preference among users? 

Finally, we know that the current Enforcement Query System (EQS) offers quite a few advanced search capabilities for MURs, which, most likely, are a significant factor in users' ability to determine the relevance of their search results. (Note: We encountered a similar situation when we investigated the Advisory Opinion search system.) Some questions here:

- **Which of the existing advanced search features are of greatest interest to users?**
    - Are any of the fields considered inessential? 
    - Are any of the fields used constantly?
- **How do users describe their ideal search experience?** Do they prefer to "cast a broad net" at first and then refine their results (like we saw with AOs), or do they prefer to compose highly complex searches from the start?
    - Note: The current system offers both options. Users can compose an advanced search at the outset of their experience, or they can conduct a keyword search and then "refine" their results. However, the option to "refine" results is actually just a replica of original "advanced search" interface. Meaning: Refinement in the existing system is not a "filtering" or "faceting" interaction, as it is on beta.fec.gov.

Given that we have all of these questions outstanding, **I think it would be wise to conduct a small research exercise with a group of advanced users at the FEC** -- individuals who would be the target users of the MUR search functionality. This would be different from the previous exercise in which we solicited generalized feedback, because these sessions would be facilitated. By facilitating a conversation with users, we will be able to probe for more information about their observations/criticisms and understand more about their motivations/needs. This will lead us to quicker answers than if we just send a list of questions or ask for another round of unmoderated responses.

For the Advisory Opinions, we conducted co-design sessions with users, which proved very helpful. I don't think it would be efficient for our timeline to try to schedule in-person co-design sessions with users. **Instead, we can schedule 45-min remote interview sessions with users.**

I can design an interview script, and we can work on recruiting the appropriate individuals, perhaps for the week following Thanksgiving.

**Comment by nickykrause** [comment]
I am tracking the research associated with this issue here: https://github.com/18F/fec-testing/issues/16

The research effort will likely generate specific issues about search results sorting, results filtering mechanisms for different enforcement matters, and (maybe) advanced search.
